pkgs/by-name/ss/ssh-openpgp-auth/generic.nix

```diff
@@ -0,0 +1,82 @@
+# This file is based upon upstream's package.nix shared among both
+# "ssh-openpgp-auth" and "sshd-openpgpg-auth"
+{ lib
+, rustPlatform
+, fetchFromGitea
+, pkg-config
+, just
+, rust-script
+, installShellFiles
+, bzip2
+, nettle
+, openssl
+, sqlite
+, stdenv
+, darwin
+, openssh
+# The arguments that cannot be empty
```

Comment:
```suggestion
# The arguments passed by package.nix
```

technically, none of the arguments can be empty, the others are just autofilled by callPackage.

Reply:
> technically, none of the arguments can be empty, the others are just autofilled by callPackage.

Thanks!